src/wallet/dump.cpp

```diff
@@ -214,6 +214,11 @@ bool CreateFromDump(const std::string& name, const fs::path& wallet_path, biling
 
             if (key == "checksum") {
                 std::vector<unsigned char> parsed_checksum = ParseHex(value);
+                if (parsed_checksum.size() != checksum.size()) {
+                    error = strprintf(_("Error: Checksum is not the correct size"));
```

Comment:
Please make this an untranslated message, it's very unlikely to appear and extremely technical.

Reply:
Done